### ISSUE TITLE
[Bug] Blog Doesn't Download if it becomes Tumbler Hidden after being added

### ISSUE THREAD

**Issue by WillScott73** [johanneszab/TumblThree#185]
There are a couple of blogs that I follow which converted to hidden days/weeks after I'd added them to TumblThree.

They no longer download. I have to remove them and then re-add them.

**Comment by Taranchuk** [comment]
Yes, I noticed it too. Converted hidden blogs just become offline as soon as I want to download them. It is because the program continues to see them as regular blogs. Here is the Index archive with such a blog, just in case, if Johaness needs such files to fix this properly.
[Index.zip](https://github.com/johanneszab/TumblThree/files/1594459/Index.zip)

**Comment by johanneszab** [comment]
As a workaround, you can manually edit the files until I've though about a useful implementation:

To change the _sampleblog_ from the Tumblr to Tumblr hidden crawler, do these steps:
1. Close TumblThree.
2. Change the file extension from .tumblr to .tmblrpriv. So, you'll have to edit these two files:
  a. sampleblog.tumblr to sampleblog.tmblrpriv
  b. sampleblog_files.tumblr to sampleblog_files.tmblrpriv
3. Open the just renamed files and change the "**BlogType**" from _0_ to _1_ in both files. Additionally, in the samepleblog.tmblpriv file change the _false_ in "**Online**" field to _true_.
4. Save both files.

The same is true if you want to do it the other way around.

The BlogType numbers are visible in [this file](https://github.com/johanneszab/TumblThree/blob/master/src/TumblThree/TumblThree.Domain/Models/BlogTypes.cs#L5), starting with zero.
